gravitee-apim-rest-api/gravitee-apim-rest-api-standalone/gravitee-apim-rest-api-standalone-distribution/src/main/resources/config/gravitee.yml

```diff
@@ -32,10 +32,10 @@
 #    path: ${gravitee.home}/dashboards
 
 # HTTP Server
-#jetty:
+jetty:
 #  IP/hostname to bind to. Default is 0.0.0.0
 #  host: 0.0.0.0
-#  port: 8083
+  port: 8083
```

Comment:
Not a good option to do this. We need to find a proper way to override this `port` when starting gravitee-node